### PR TITLE
New AdminAPI format POST tests

### PR DESCRIPTION
**Pull request by austin-king**
## High Level Overview of Change

POST test for new AdminAPI format

### Context of Change

Final PR adding tests to support this format

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [x] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After

Just a new test in `postUsers.test.ts`

## Future Tasks

Once this is approved I'll merge it into my other test PR. I'll pull that down, clean it up (I believe it will need linting), push it up then open a PR to request merging into @dino-rodriguez 's logic branch. At that point he'll be able to run the tests against his logic locally to make sure everything is working, he'll push up the PR and we'll know that functionality is working as expected.

### CODE REVIEW

File: test/integration/e2e/admin-api/postUsers.test.ts [hunk 1]
```diff
@@ -172,6 +174,67 @@ describe('E2E - adminApiRouter - POST /users', function (): void {
       })
   })
 
+  it('Returns a 201 when creating a new user with the CLI format', function (done): void {
+    // GIVEN a user with a PayID known to not exist on the PayID service
+    const payId = 'johnnywalker$emptybottles.com'
```
Comment (dino-rodriguez) [inline test/integration/e2e/admin-api/postUsers.test.ts:179]: Too many tests are known to drive a man to the bottle.

File: test/integration/e2e/admin-api/postUsers.test.ts [hunk 2]
```diff
@@ -172,6 +174,67 @@ describe('E2E - adminApiRouter - POST /users', function (): void {
       })
   })
 
+  it('Returns a 201 when creating a new user with the CLI format', function (done): void {
```
Comment (dino-rodriguez) [inline test/integration/e2e/admin-api/postUsers.test.ts:177]: nit : Should this be `CLI` or something like `canonical` or `Public API` ?

Comment (dino-rodriguez) [inline test/integration/e2e/admin-api/postUsers.test.ts:177]: nit : Should this be `CLI` or something like `canonical` or `Public API` ?

File: test/integration/e2e/admin-api/postUsers.test.ts (hunk 1 above)
Comment (dino-rodriguez) [inline test/integration/e2e/admin-api/postUsers.test.ts:179]: Too many tests are known to drive a man to the bottle.

**Comment by austin-king** [comment]
Cool
CLI -> canonical

> Looks like a good start! I would take a look at some of the logic in the POST PR to make sure we are testing cases involving identity key validation as well.

And good point, I did this w/o looking at your branch so today I'll spend some time looking at other units of logic to test from your branch

**Comment by austin-king** [comment]
Cool
CLI -> canonical

> Looks like a good start! I would take a look at some of the logic in the POST PR to make sure we are testing cases involving identity key validation as well.

And good point, I did this w/o looking at your branch so today I'll spend some time looking at other units of logic to test from your branch

File: test/integration/e2e/admin-api/postUsers.test.ts (hunk 1 above)
Comment (hbergren) [reply test/integration/e2e/admin-api/postUsers.test.ts:179]: We're getting really creative with PayID names now 😄

**Pull request merged**